### ISSUE TITLE
Transform flyfile AST

### ISSUE THREAD

**Issue by hzlmn** [lukeed/fly-esnext#8]
Seems like right now, `fly-esnext` just replace some syntax constructions using regular expressions. It would be nice if it use `babylon` or `uglify` etc. to transfrom file AST, it will be more correct behavior for syntax transformations.

**Comment by lukeed** [comment]
Yeah, correct. It's just regex replacements cuz they're the fastest and I only intended to support a few things. What are you missing?

**Comment by hzlmn** [comment]
@lukeed I am not missing something, it just could be confusion cuz if you write your export like
```js
export      function test() {}
```
it won't be transformed :smile: . Also as i know global regexps not cheap and you scan source at least 4 times, so maybe it will be more efficient to transform it from AST. Also adding `import` transform will be easier.

**Comment by lukeed** [comment]
Lol, right. I assume normal whitespace variants ;-)

What do you mean about scanning source 4 times? I only scan once, during Fly boot up. It's actually MUCH faster than a babylon-parser (or any AST I tried).
